### PR TITLE
chore: commands are checking that Function is initialized

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

Fixes: #161

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ kn faas build [-i <image> -r <registry> -p <path>]
 
 ## `run`
 
-Runs the Function project locally in the container. If a container has not yet been created, prompts the user to run `faas build`. Note: there is no option to specify a path to the project.
+Runs the Function project locally in the container. If a container has not yet been created, prompts the user to run `faas build`.  The user may specify a path to the project directory using the `--path` or `-p` flag.
```
Comment (lance) [inline docs/commands.md:39]: > If a container has not yet been created, prompts the user to run `faas build`. 

Does it? I don't see where this is happening - I could be missing something obvious of course.

**Review by lance — COMMENTED** [review]
Looks good - just one question in the comments.

File: docs/commands.md (hunk 1 above)
Comment (zroubalik) [reply docs/commands.md:39]: https://github.com/boson-project/faas/blob/e425c8f08183b333e56d5d3cfe74fc9e85a6c903/docker/runner.go#L33

Comment (lance) [reply docs/commands.md:39]: Please ignore my comments in the future! LOL

**Review by lance — APPROVED** [review]

**Pull request merged**